plugins/inputs/win_services/win_services_test.go

```diff
@@ -0,0 +1,101 @@
+// +build windows
+
+//this test must be run under administrator account
+package win_services
+
+import (
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+    "golang.org/x/sys/windows/svc/mgr"
+)
+
+var InvalidServices = []string{"XYZ1@", "ZYZ@", "SDF_@#"}
+var KnownServices = []string{"LanmanServer", "TermService"}
+
+func TestList(t *testing.T) {
+	services, err := listServices(KnownServices)
```

Comment:
Chronograf uses a nice pattern for mocks that can handle this.  I wrote an example and added it to this gist https://gist.github.com/danielnelson/79908f0bc7145d3feb7a91e0ef56d88c